test/e2e/suites/e2e/e2e_test.go

```diff
@@ -142,7 +142,7 @@ var _ = Describe("e2e tests", func() {
 		})
 	})
 
-	Describe("Workload cluster (external cloud provider)", func() {
+	FDescribe("Workload cluster (external cloud provider)", func() {
```

Comment:
Yes, I'm still using it. The external CCM case is still broken for some reason I haven't worked out yet. I was having trouble running the tests locally because unfortunately OpenStack CCM still uses docker.io, and I haven't worked out how to pass a pull secret into the tests yet.

Incidentally, how do you feel about the change which only dumps machines during cleanup? The problem I find with the machine ticker is that when your test fails you don't see the state it was in when it failed; you see the state it was in the last time the machine ticker ran, which was most likely while it was being deleted. I'd like to run the resource dump at exactly 2 points:

1. Before cleanup
2. Optionally, if cluster delete fails or times out

I think anything else is just overwriting the logs you're actually looking for.

Reply:
Sorry for my late response. I'm not yet very familiar with the e2e tests for CAPO.
As per your description it sounds very good to just dump when really needed - probably even the second case would be sufficient. So when the delete fails or times out.